.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "loopback",
+  "rules": {
+    "max-len": ["error", 100, 4, {
+      "ignoreComments": true,
+      "ignoreUrls": true,
+      "ignorePattern": "^\\s*var\\s.+=\\s*(require\\s*\\()|(/)"
+    }]
+  }
+}
```

Comment:
We should probably just use JSON on second thought. It would be less confusing for community too. I haven't really seen anyone use the yaml version in node projects.